scapy/packet.py

```diff
@@ -1459,9 +1459,11 @@ def fuzz(p, _inplace=0):
                     print "fuzzing", repr(r)
                     fuzz(r, _inplace=1)
             elif f.default is not None:
-                rnd = f.randval()
-                if rnd is not None:
-                    q.default_fields[f.name] = rnd
+                is_cond_field = isinstance(f, ConditionalField)
+                if not is_cond_field or (is_cond_field and f._evalcond(q)):
```

Comment:
Taking advantage of Python lazy evaluation, you can make these two lines easier to read as:
```Python
    if not isinstance(f, ConditionalField) or f._evalcond(g):
```